src/pages/PlayerStats/components/NavigationBar.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { styled } from "linaria/react";
+import { Button, Link } from "../../../primitives";
+import { ListIcon, SearchIcon } from "../../../ui-icons";
+
+const Nav = styled.nav`
+  display: flex;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  padding: var(--spacing-base);
+  z-index: var(--depth-layer2);
+  justify-content: ${({ hasMoreLegends }) =>
+    hasMoreLegends ? "space-between" : "flex-end"};
+
+  ${Link}, ${Button} {
+    width: 1.25rem;
+  }
+
+  @media (min-width: 1024px) {
```

Comment:
The breakpoint value is used in multiple components, convert to css variable for better maintainability

Reply:
Incorrect usage of css variables - [SO](https://stackoverflow.com/questions/40722882/css-native-variables-not-working-in-media-queries)